stable/ipfs/templates/service.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ if .Values.service.nameOverride }} {{ .Values.service.nameOverride }} {{ else }} {{ template "fullname" . }} {{ end }}
```

Comment:
Why would you want to do that? This should just be `{{ template "fullname" . }}`.